Makefile

```diff
@@ -24,10 +24,10 @@ run:
 
 clean:
 	rm image/cli/install-ansible/ibm-mas_devops.tar.gz
-	rm image/cli/bin/templates/ibm-mas-tekton.yaml
+	rm image/cli/bin/templates/ibm-mas-tekton.yml
 
 create:
-	oc apply -f tmp/deployment.yaml
+	oc apply -f tmp/deployment.yml
```

Comment:
As above, yaml (openshift/kubernetes) versus yml (ansible), don't think we really want to change this.

Reply:
Done. See comment above